modules/cudf/src/data_frame.ts

```diff
@@ -884,4 +884,35 @@ export class DataFrame<T extends TypeMap = any> {
     });
     return new DataFrame(series_map);
   }
+
+  /**
+   * Creates a DataFrame of `BOOL8` Series where `true` indicates the value is `null` or `NaN` and
+   * `false` indicates the value is valid.
+   *
+   * @returns DataFrame<Bool8> with series of `BOOL8` elements with `true` representing `null` or
+   * `NaN` values
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series, Int32, Float32}  from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *  a: Series.new({type: new Int32, data: [0, 1, 2, 3, 4, 4]}),
+   *  b: Series.new({type: new Float32, data: [0, NaN, 2, 3, 4, 4]})
+   * });
+   *
+   * df.isNull()
+   * ```
+   */
+  isNull(): DataFrame<T> {
```

Comment:
I saw that `subset` was an optional param for a lot of the other dataframe operations, do we want to include this for basically all dataframe methods (where it makes sense?)

Reply:
it doesn't look like Python's [`DataFrame.isnull()`](https://github.com/rapidsai/cudf/blob/2c70f1d5084c12bd04f676b95d5f20cb53f67250/python/cudf/cudf/core/frame.py#L2264-L2339) accepts a subset argument, so we shouldn't need one here. 

The `dropNulls` and `dropNaN` methods accept a subset, i.e. a list of column names to consider when dropping rows. This allows the user to exclude certain columns from consideration when dropping rows that contain null/NaN values:

```js
console.log(...new DataFrame({
  a: Series.new([1, null, 2]),
  b: Series.new([null, 1, 2]),
}).dropNulls(1, 1, ['a']).toArrow()]);
// [
//   { a: 1, b: null },
//   { a: 2, b: 2 }
// ]

```

Reply:
Ah, the one example I looked at was `nansToNulls` which is defined above, it takes in a `subset` even though the docs mention it shouldn't. Shall I remove the arg?
https://docs.rapids.ai/api/cudf/nightly/api.html#cudf.core.dataframe.DataFrame.nans_to_nulls

Reply:
Spoke about this over slack, leaving as is.